manuscript/book.md

```diff
@@ -39,21 +39,21 @@ for (let i = 0; i < names.length; i++) {
 }
 ```
 
-And now with the `.map()` method:
+Same example but with the `.map()` method instead of `for` loop:
 
 ```js
 const names = ['Bilbo Baggins', 'Gandalf', 'Gollum'];
 const kebabNames = names.map(name => _.kebabCase(name));
 ```
 
-We can shorten it even more if our processing function accepts only one argument, and [kebabCase from Lodash](https://lodash.com/docs#kebabCase) does:
+We can shorten it even more if our processing function accepts the value as first argument, [kebabCase from Lodash](https://lodash.com/docs#kebabCase) does:
```

Comment:
```suggestion
We can shorten it even more by passing reference to the callback function if it accepts only one argument, so the array index, passed by the `.map()` method as a second argument, does not affect the result. 
Like [kebabCase](https://lodash.com/docs#kebabCase) from Lodash:
```

Reply:
I don't understand this sentence at all ;-/

⌛️ Should think how to better explain it.

Reply:
Indeed this is a complicated sentence. I think splitting this conditional sentence into two sentences would make it easier to consume. Something like:
`If <requirements for the improvement> are met, you can <do the improvement>` style